apps/backend/src/app/modules/admin/admin.controller.ts

```diff
@@ -63,9 +67,36 @@ export class AdminController {
     private readonly mapsService: MapsService,
     private readonly mapReviewService: MapReviewService,
     private readonly usersService: UsersService,
-    private readonly adminActivityService: AdminActivityService
+    private readonly adminActivityService: AdminActivityService,
+    private readonly killswitchService: KillswitchService
   ) {}
 
+  @Patch('/killswitch')
+  @HttpCode(HttpStatus.NO_CONTENT)
+  @ApiOperation({ summary: 'Update active killswitches' })
+  @ApiNoContentResponse({
+    description: 'The switches were updated successfully'
+  })
+  @ApiBadRequestResponse({ description: 'Invalid switches type' })
+  @ApiBody({
+    type: Object,
```

Comment:
Fine to not do a schema for this (annoying to do) but maybe update description in line below to say e.g. `Object of type Record<KillswitchType, boolean>`